codegen/swagger-codegen.adoc

```diff
@@ -0,0 +1,65 @@
+---
+permalink: /codegen_swagger_codegen/
+---
+= Using Swagger Codegen in {RAS}
+:imagesdir: ../images/codegen
+:linkattrs:
+:docinfo2:
+:RAS: RepreZen API Studio 
+:RZ: RepreZen
+
+In {%link cg_intro%} we discuss GenTemplates and GenTargets in the context of RAPID-ML models. But
+as you may know, {RAS} is a great platform for working with Swagger -- a.k.a. OpenAPI -- API
+specifications. This includes full access to the
+link:https://github.com/swagger-api/swagger-codegen[Swagger Codegen] framework.
+
+If you've used Swagger Codegen (SCG), you'll know that the implementation comes with several
+out-of-the-box modules that generate documentation, client libraries for a variety of languages, and
+service scaffolding, also geared toward a variety of languages and underlying frameworks. You'll be
+glad to know that all of these standard SCG modules are available as GenTemplates in {RAS}.
+
+== Model Type Determines GenTemplates
```

Comment:
@andylowry , just a heads-up that we will want to change this, making SCG templates available for RAPID-ML files.  This will use a RAPID-->Swagger GenTarget as a prerequisite.  You don't need to make note of that in this doc right now, but we'll want to address this fairly soon.  Some of it depends on whether our current or future adopters (hopefully near-future, through SBC Fintech) want to use RAPID-ML.  RAPID-ML looks a lot more viable if it has pseudo-direct access to all of the SCG modules.